### PR TITLE
Replace arrow entities for glyphicon + template + libs snippets

### PR DESCRIPTION
**Pull request by Azaret**
I'll start this PR for the replacement of the arrow entities by the glyphicon icons.
With using some `templates` option so anyone can easily use their favorite icons libs.
I'll ensure that no templates break the core structure as @eternicode wished.
Linked issues/PRs : #610 #833 #834 #1007

- [x] Fix the `click` event
- [x] Add `templates` options
- [x] Docs
- [x] Qunit tests
- [x] Libs Snippets

### CODE REVIEW

**Comment by Azaret** [comment]
@acrobat @hebbet I let you check this :)

File: js/bootstrap-datepicker.js [hunk 1]
```diff
@@ -1104,6 +1131,108 @@
...
+					if (this.o.minViewMode === 2){
+						this._setDate(UTCDate(year, month, day));
+					}
+					this.showMode(-1);
+					this.fill();
+				}
+			}
+
+			/*
+			 * Commenting old click event to avoid future conflict with decade picker
```
Comment (acrobat) [inline js/bootstrap-datepicker.js]: Please don't comment code, just remove it if it's not necessary. You can easily check earlier revisions with git!

Comment (Azaret) [inline js/bootstrap-datepicker.js]: I planned to remove it when merging back the master

**Comment by hebbet** [comment]
@Azaret could you rebase to current master?

**Comment by acrobat** [comment]
ping @Azaret

**Comment by Azaret** [comment]
Gosh I've like 2 billions ms lag...
Sorry I've a bit busy lately and I had no time for breaking this plugin :D 
Anywhosies, I'll look into this, like this weekend, and the other stuff around here...

**Comment by Azaret** [comment]
Had to make a new branch for this one too, changes were too complex.

**Pull request closed without merging**